Problem with docker

# Docker command finishes with an error 
Hi. I followed your steps presented in the tutorial. When I run a command: docker-compose up, I get an error:

```bash
-- Installation directory: /workspaces/ros2_fprime_bridge/fprime_ws/spring/build-artifacts
-- Performing CMake source prescan
CMake Error at /usr/share/cmake-3.22/Modules/FindProtobuf.cmake:272 (message):
  Error: PROTOBUF_GENERATE_CPP() called without any proto files
Call Stack (most recent call first):
  ros2_bridge/CMakeLists.txt:20 (PROTOBUF_GENERATE_CPP)


CMake Error at /app/fprime/cmake/prescan.cmake:127 (message):
-- Configuring incomplete, errors occurred!
See also "/workspaces/ros2_fprime_bridge/fprime_ws/spring/build-fprime-automatic-native/CMakeFiles/CMakeOutput.log".
  Failed to run prescan, cannot continue.
Call Stack (most recent call first):
  /app/fprime/cmake/FPrime.cmake:64 (perform_prescan)
  CMakeLists.txt:7 (include)


[ERROR] CMake erred with return code 1. Partial build cache remains. Run purge to clean-up.
The command '/bin/sh -c . /tmp/fprime-venv/bin/activate &&      cd fprime_ws/spring &&      fprime-util generate &&      fprime-util build' returned a non-zero code: 1
ERROR: Service 'foxglove_adapter' failed to build : Build failed
```
Any idea what I can do in order to fix this problem?

--- a/Dockerfile
+++ b/Dockerfile
@@ -51,7 +51,13 @@ RUN apt-get update && \
 WORKDIR /workspaces/ros2_fprime_bridge
 
 # copying workspaces to the docker
+COPY examples/ examples/
 COPY common/ common/
+
+# generate protobuf files
+RUN cd common/autogen && \
+    python3 compile.py spring_defs.json && \
+    protoc -I=schemas/spring/proto3 --cpp_out=./schemas/spring/proto3 spring.proto
 
 #compiling fprime workspace
 COPY fprime_ws/ fprime_ws/
@@ -60,8 +66,6 @@ RUN  git clone https://github.com/nasa/fprime.git /app/fprime && \
      git reset --hard a56426adbb888ce4f5a8c6a2be3071a25b11da16
 RUN  . /tmp/fprime-venv/bin/activate && python3 -m pip install -U -r /app/fprime/requirements.txt
 RUN  rm -rf fprime_ws/spring/build-artifacts fprime_ws/spring/build-fprime-automatic-native
-
-COPY examples/ examples/
 
 RUN  . /tmp/fprime-venv/bin/activate && \
      cd fprime_ws/spring && \

